docs/replication.md

```diff
@@ -236,10 +258,35 @@ multiCluster:
 
 Apply changes as usual with the `kubectl apply -f deploy/cr.yaml` command.
 
+!!! note
+
+    If you want to enable multi-cluster services in a new cluster, we
+    recommended deploying the cluster first with `multiCluster.enabled` set to
+    `false` and enable it after replset is initialized.
```

Comment:
why? What will happen if I do otherwise? Again, the risk is not explained here.

Reply:
well, the problem is replset initialization fails **sporadically** if you enable MCS from the start. I don't exactly know why to be honest. Can we say something like "Having MCS enabled from the start prone to errors on replset initialization"?

Reply:
I think we can

Reply:
https://github.com/percona/k8spsmdb-docs/pull/39/commits/a4603e29725cbec78eaa928f4c519273e0de34d8